docs/tutorial.rst

```diff
@@ -0,0 +1,1606 @@
+    $ cd psamm-tutorial
+
+Now the user should be in the ``psamm-tutorial`` folder and should see the
+following folders:
+
+.. code-block:: shell
+
+    additional_files/
+    E_coli_sbml/
+    E_coli_json/
+    E_coli_excel/
+
+These files include all of the files that will be needed to run the tutorial.
+
+2) `PSAMM` Installation
+_________________________
+
+`PSAMM` can be installed using the Python package installer pip. We recommend
+that all installations be performed under a virtual python environment. Major
+programs and dependencies include: ``psamm-model``, which supports model checking,
+model simulation, and model exports; LP solvers (e.g. Cplex, Gurobi, QSopt_ex),
+which supports the solution of linear programming problems; ``psamm-import``, which
+supports the import of models in SBML, JSON, and EXCEL formats.
+
+
+
+A) Setting up a Virtual Python Environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is recommended that the `PSAMM` software and dependencies should be installed
+under a virtual python environment. This can be done by utilizing the Virtualenv
+software. Virtualenv will set up a python environment that permits the users to
+install python packages in a local directory that will not interfere with other
+programs in the global python. The virtual environment can be set up at any
+local directory that is writable to the user. For example, here we will set up
+the virtual environment under the main directory of this `PSAMM` tutorial. First,
+run the following command if the user is not in the psamm-tutorial folder:
+
+.. code-block:: shell
+
+    $ cd <PATH>/psamm-tutorial
+
+n this command, <PATH> should be substituted by the directory path to where
+you stored the psamm-tutorial. This will change your current directory to the
+psamm-tutorial directory. Then, you should start a virtual environment in the
+psamm-tutorial directory:
+
+.. code-block:: shell
+
+    $ virtualenv psamm-env
+
+That will set up the virtual environment in a folder called psamm-env/. The
+next step is to activate the virtual environment so that the python that is
+being used will be the one that is in the virtualenv. To do this use the
+following command:
+
+.. code-block:: shell
+
+    $ source psamm-env/bin/activate
+
+This will change your command prompt to the following:
+
+.. code-block:: shell
+
+    (psamm-env) $
+
+This indicates that the virtual environment is activated, and any installation
+of python programs will now be installed in the virtual environment. It is
+important to note that when the user leaves the environment and returns at a
+later time they will have to reactivate the environment (use the ``source`` command
+above) to be able to use any packages installed in it.
+
+.. note:: For windows users the virtual environment is installed in a different
+file structure. The activate script on these systems will reside in a scripts
+folder. To activate the environment on these systems use the command:
+
```

Comment:
In this particular case it also seems like the following code block should go into the note but I don't know if that is actually possible.